.github/workflows/osrm-backend.yml

```diff
@@ -715,6 +715,7 @@ jobs:
         pushd ${OSRM_BUILD_DIR}
```

Comment:
Baseline on my M1 Pro/32 gb ram:
<img width="905" alt="initial" src="https://user-images.githubusercontent.com/266271/193300237-66b57683-ac23-4324-81ed-bacc11170bb9.png">

Reply:
"string" here is `std::stringstream ss; ... = ss.str()`